libs/fixtures/data/electionGeneral/ui-strings.json

```diff
@@ -1748,7 +1748,7 @@
     "hmpbOfficialProvisionalBallot": "Boleta provisional oficial",
     "hmpbPage": "Página",
     "hmpbPageIntentionallyBlank": "Esta página se dejó en blanco intencionalmente.",
-    "hmpbSampleAbsenteeBallot": "Ejemplo de boleta de voto en ausencia",
+    "hmpbSampleAbsenteeBallot": "Modelo de boleta de voto ausente",
```

Comment:
Making it consistent with the other sample ballot translations (and a bit shorter to avoid line wrapping)